examples/combined_ODNP.py

```diff
@@ -382,3 +390,4 @@
 with h5py.File(os.path.join(target_directory, filename), "a") as f:
     log_grp = f.create_group("log")
     hdf_save_dict_to_group(log_grp, this_log.__getstate__())
+print('*'*30+'\n'+'\n'.join(final_log))
```

Comment:
I don't like this input thing -- rather save the errors for the end.  It's faster.

Reply:
sounds good

Reply:
what are the astrix though?

Reply:
oh shoot yes I know AB used to do that a lot!